tests/old_suite/runtests.py

```diff
@@ -560,11 +560,12 @@ def _generic_test_function(self):
         self.assertTrue(b.test_building(),
                 msg='Build of %s failed.' % self.test_name)
 
+        """"
         okay, msg = b.test_logs()
         if not okay:
             self.fail('Matching .toc of %s failed.\n\n%s' %
                       (self.test_name, msg))
-
+        """
```

Comment:
Unused code should be removed. Otherwise we are carrying around legacy for years. If it shows up the be required, we can restore it from the VCS.